Misc/NEWS.d/next/Library/2020-08-10-15-06-55.bpo-21041.cYz1eL.rst

```diff
@@ -0,0 +1 @@
+Pathlib path.parents now support negative indexing. Patch contributed by Yaroslav Pankovych.
```

Comment:
```suggestion
:meth:`~pathlib.Path.parents` now supports negative indexing. Patch contributed by Yaroslav Pankovych.
```

Alternatively you can drop the `~` to have this render as `pathlib.Path.parents`. If you can render the docs and check that this link works properly, that would be good, because I didn't test it myself 😅

Reply:
Oh sure, I'll render it.